src/index.tsx

```diff
@@ -1,7 +1,7 @@
 import './i18n';
 
 import { Web3ReactProvider } from '@web3-react/core';
-import { StrictMode } from 'react';
```

Comment:
Strict mode helps with identifying potentional lifecycle issues. I would keep it.